### DISCUSSION TITLE
No images when presenter mode is accessed remotely.

### DISCUSSION THREAD

**Discussion by fabiowmm** [slidevjs/slidev#1833] [Q&A]
Is this a known issue? If I access the presentation remotely as a viewer all figures render perfectly, but if I try to access in presenter mode all images are missing. Thanks in advance!

**Comment by kermanx** [comment]
Not known yet. This docs may help: https://sli.dev/guide/faq#assets-handling

**Reply by fabiowmm** [reply]
This reference is for builds. I'm not building anything, just presenting my slides in a Mac and remote controlling from an iPad. If I access the Mac web server from the iPad as a regular viewer, images load just right. The problem is when I access as presenter to control it.

**Comment by wicked539** [comment]
Did you ever figure this out? Still seems to be broken (Slidev v51.0.2 on MacOS, running `npm run dev`, and then opening `http://localhost:3030/presenter/` in browser.

**Comment by wicked539** [comment]
Problem was the image URL. For the image to be visible in presenter view, it needs to be either absolute (eg, `https://foo.bar/images/dog.png`), or relative _and starting with a slash_ (eg, `/images/dog.png`)
